Rodrigo Albornoz - MongoDb - Exercício 01 - Resolvido

albornozrodrigo opened the pull request:
Exercício 01 do módulo de MongoDb do curso BeMEAN.

albornozrodrigo committed:
Create class-01-resolved-xchilenox-RodrigoAlbornoz.md

DouglasHennrich commented:
Está tudo certo no exercício, MENOS, o nome do arquivo =(

vc tem que separar o seu nome com hífens, que nem está no README.md no repositório

albornozrodrigo committed:
Rodrigo Albornoz - MongoDB - Exercício 01 - Arquivo renomeado

Rename class-01-resolved-xchilenox-RodrigoAlbornoz.md to class-01-resolve-xchilenox-Rodrigo-Albornoz.md

xchilenox commented:
Beleza, já renomeei o arquivo, os próximos também irão no padrão.

DouglasHennrich commented:
Boa
